extra_foam/config.py

```diff
@@ -102,6 +102,11 @@ class PipelineSlowPolicy(IntEnum):
     WAIT = 1
 
 
+class CaliOffsetPolicy(IntEnum):
```

Comment:
`CalOffsetPolicy` if Cali is calibration? or `CalibrationOffsetPolicy`

Reply:
Yes! Will change to the more verbose one.

Reply:
Changed! Thank you!